devito/arch/compiler.py

```diff
@@ -456,6 +456,14 @@ def __init__(self, *args, **kwargs):
         else:
             self.cflags += ['-march=native']
 
+        # For mpi, mpicc is compiled against amdclang not aompcc, so need the flags back.
```

Comment:
MPI, not mpi

but aside from this nitpick, this is necessary for our dockerfile right? 
should we not rather just add an AMDClangCompiler class?